content/enum/PostFlag.md

```diff
@@ -0,0 +1,126 @@
+**Project NewCaw** \
+Author: Frederick Schenk
+
+# Content/PostFlag
+
+![API: Twitter 1](https://img.shields.io/badge/API-Twitter%201-lightgrey?style=flat-square) ![API: Twitter 2](https://img.shields.io/badge/API-Twitter%202-blue?style=flat-square) ![API: Mastodon](https://img.shields.io/badge/API-Mastodon-purple?style=flat-square)
+
+```c#
+public enum PostFlag
+```
+
+Defines different states for an [`Post`](../class/Post.md).
+
+## Values
+
+### `DELETED`
+
+![API: Internal](https://img.shields.io/badge/API-Internal-green?style=flat-square)\
+
+This Post was recently deleted, but is still in Memory.
+
+### `VIEWED`
+
+![API: Internal](https://img.shields.io/badge/API-Internal-green?style=flat-square)\
+
+This Post was seen by the user. Used to determine where to place the viewport of the timeline viewer.
+
+### `LIKED`
+
+![API: Twitter 1](https://img.shields.io/badge/API-Twitter%201-lightgrey?style=flat-square) ![API: Twitter 2 (Not yet)](https://img.shields.io/badge/API-Twitter%202%20(Not%20yet)-darkred?style=flat-square) ![API: Mastodon](https://img.shields.io/badge/API-Mastodon-purple?style=flat-square)
+
+This Post was liked by the active user.
+
+| API   | Twitter 1   | Twitter 2 | Mastodon    |
+| ----- | ----------- | --------- | ----------- |
+| Field | `favorited` |           | `favorited` |
+| Type  | bool        |           | bool        |
+
+### `REPOSTED`
+
+![API: Twitter 1](https://img.shields.io/badge/API-Twitter%201-lightgrey?style=flat-square) ![API: Twitter 2 (Not yet)](https://img.shields.io/badge/API-Twitter%202%20(Not%20yet)-darkred?style=flat-square) ![API: Mastodon](https://img.shields.io/badge/API-Mastodon-purple?style=flat-square)
+
+This Post was reposted by the active user.
+
+| API   | Twitter 1   | Twitter 2 | Mastodon    |
+| ----- | ----------- | --------- | ----------- |
+| Field | `retweeted` |           | `reblogged` |
+| Type  | bool        |           | bool        |
+
+### `REPLIED`
+
+![API: Internal](https://img.shields.io/badge/API-Internal-green?style=flat-square)\
+
+This Post has an reply from the active user.
+
+### `HIDDEN_ON_TIMELINES`
+
+![API: Mastodon](https://img.shields.io/badge/API-Mastodon-purple?style=flat-square)\
+
+This Post is set to not be shown on timelines.
+
+| API   | Mastodon              |
+| ----- | --------------------- |
+| Field | `visibility/unlisted` |
+| Type  | string enum           |
+
+### `HIDDEN_UNLESS_FOLLOWER`
```

Comment:
I'm guessing this is more granular than setting a protected account.

Reply:
Indeed it is.